src/vue/forms/InvestForm.vue

```diff
@@ -163,7 +163,15 @@
         icon-name="alert-circle"
         :title="'invest-form.insufficient-balance-title' | globalize"
         :message="'invest-form.insufficient-balance-desc' | globalize"
-      />
+      >
+        <router-link
+          :to="vueRoutes.movements"
+          tag="button"
```

Comment:
Add `type="button"`. `<button>` as default has a `submit` type, so in this case, click on this button will submit the form